pkg/steps/clusterinstall/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+  - aaronlevy
+  - abhinavdahiya
+  - crawford
+  - rajatchopra
+  - smarterclayton
+  - staebler
+  - wking
+installer-reviewers:
```

Comment:
Don't think this will do anything in this repo

Reply:
Aargh, sloppy work. Sorry :(